jobs/e2e_node/containerd/config-systemd.toml

```diff
@@ -17,7 +17,6 @@ oom_score = -999
 [plugins."io.containerd.grpc.v1.cri".cni]
   bin_dir = "/home/containerd/"
   conf_dir = "/etc/cni/net.d"
-  conf_template = "/home/containerd/cni.template"
```

Comment:
uh, but we still need to configure CNI, so something else will have to do this?

just removing it is not an appropriate migration